agent/agentssh/agentssh_internal_test.go

```diff
@@ -60,7 +61,8 @@ func Test_sessionStart_orphan(t *testing.T) {
 		// we don't really care what the error is here.  In the larger scenario,
 		// the client has disconnected, so we can't return any error information
 		// to them.
-		_ = s.startPTYSession(sess, cmd, ptyInfo, windowSize)
+		m := metricsForSession(s.metrics.sessions, "ssh")
```

Comment:
this should be above the comment so that it's more clear the comment is referring to the `s.startPTYSession` line.

Reply:
Fixed